plugins/focus-tracker/style.less

```diff
@@ -0,0 +1,9 @@
+@import "../../less/variables.less";
+
+.tota11y-outlined {
+	outline: 2px solid fadein(@highlightColor, 100%);
+}
+
+.tota11y-was-focused {
+  outline: 2px dashed fadein(@highlightColor, 80%);
```

Comment:
Previously focused elements receive a slightly lighter, dashed version of the outline we apply while something is focused.